setup.py

```diff
@@ -70,7 +70,7 @@ def build(cls):
         try:
             import sass
         except ModuleNotFoundError:
-            pip.main(['install', 'libsass'])
+            check_call([sys.executable, '-m', 'pip', 'install', 'libsass'])
```

Comment:
yes, but you guy left it out of the requirements.txt because it is only needed if one tries to build the package. @cemsbr and @diraol said to leave it that way, so I left it..